tooling/cli/Cargo.toml

```
@@ -92,6 +92,7 @@ libc = "0.2"
 lto = true
 
 [features]
-default = [ "tauri-bundler/default" ]
+default = [ "tauri-bundler/rustls" ]
```

i guess both work but i still feel like this should be this right?
```suggestion
default = [ "rustls" ]
```

---

Yeah, thanks (damn I didn't see this as a suggestion and manually commited it)